plugin-manager/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+use agave_geyser_plugin_interface::geyser_plugin_interface::GeyserPlugin;
+use solana_sdk::{
+    account::AccountSharedData, clock::Slot, pubkey::Pubkey, transaction::SanitizedTransaction,
+};
+
+use crate::accounts_update_notifier::AccountsUpdateNotifier;
+
+mod accounts_update_notifier;
+
+#[derive(Default)]
+pub struct GeyserPluginService {
+    accounts_update_notifier: Option<AccountsUpdateNotifier>,
+}
+
+impl GeyserPluginService {
+    pub fn set_plugin(&mut self, plugin: Box<dyn GeyserPlugin>) {
+        if plugin.account_data_notifications_enabled() {
+            self.accounts_update_notifier = Some(AccountsUpdateNotifier::new(plugin));
+        }
+    }
+
+    pub fn notify_account_update(
+        &self,
+        slot: Slot,
+        txn: &Option<&SanitizedTransaction>,
+        account: &AccountSharedData,
+        pubkey: &Pubkey,
+    ) {
+        if let Some(notifier) = self.accounts_update_notifier.as_ref() {
+            notifier.notify_account_update(
+                slot, account, txn, pubkey, 1, // NOTE: hardcoded write_version
```

Comment:
I don't know much the impact but maybe it's better to be close than the 'real example' behaviour, is it hard to add it?

Reply:
the order of updates is not guaranteed and this is a common situation when plugin developers use that field to store the updates properly.

e.g. you're receiving updates with write-version in the next order:
`1 - 3 - 2`, after reading the update with `v3` you will disrespect changes in the next update (with `v2`)


no, this is not a big deal, the only thing that we have to keep in mind is that we're moving closer to the real Solana-runtime instead of keeping everything as light as possible.

Reply:
Ok let's keep it hardcoded